xsl/xccdf-report-impl.xsl

```diff
@@ -452,12 +452,15 @@ Authors:
     </xsl:for-each>
 </xsl:template>
 
-<xsl:template name="convert-url-to-name">
+<xsl:template name="convert-reference-url-to-name">
     <xsl:param name="href"/>
     <xsl:choose>
         <xsl:when test="starts-with($href, 'http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53')">
             <xsl:text>NIST SP 800-53 ID</xsl:text>
         </xsl:when>
+        <xsl:when test="starts-with($href, 'http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-171')">
+            <xsl:text>NIST SP 800-171 ID</xsl:text>
```

Comment:
We're inconsistent with naming. Some are appended with ``ID``. aka ``DISA CCI`` vs ``NIST SP 800-171 ID``.

Suggest we drop the ``ID``